app.ts

```diff
@@ -15,14 +12,16 @@ import favicon from 'serve-favicon';
 import raven from 'raven';
 import router from './app/routes';
 import User from './app/models/user';
-import env from './env';
+import env from "./env";
+const app = express();
+// eslint-disable-next-line
```

Comment:
No. Unless you have a really really good reason for this, you have to remove this and fix the lint issue.